regtests/setup.sh

```diff
@@ -105,13 +105,10 @@ spark.jars.packages org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:1.5.2,org.
 spark.hadoop.fs.s3.impl org.apache.hadoop.fs.s3a.S3AFileSystem
 spark.hadoop.fs.AbstractFileSystem.s3.impl org.apache.hadoop.fs.s3a.S3A
 spark.sql.variable.substitute true
-
```

Comment:
Unrelated whitespace change. I think we should probably also keep this a little split up for clarity

Deps

Hadoop

Driver

Sql - Session

Sql - Catalog

Reply:
^ This can be saved for a later cleanup if someone else who is as nit-picky as me cares :)

Reply:
Removed the change.